Procfile

```diff
@@ -1,3 +1,3 @@
-web: app/app.py --config=app/config.py
-db: mongod --dbpath data/db
+web: python app/app.py --config=app/config.py
+db: mongod --dbpath c:\mongodb\data
```

Comment:
Rather than commit this change, let's document it in the README.md.  Supper awesome you got it running on Windows!